### ISSUE TITLE
Translations: Confusing some entities/comments upside down

### ISSUE THREAD

**Issue by yfdyh000** [qsniyg/maxurl#1020]
I noticed that some important strings/comments were upside down in the .po files, which was detrimental to the localization process.

such as:
`$language_native$`, `$description$`.
`category_redirection`, etc.
`saved_refresh_target`, etc.
`rotate_left_btn`, etc.

The translator may make the wrong decision (like keep strings) if does not notice that the content/translator's comments are reversed.
I look forward to this being solved. However, there may be technical hurdles, and it will take effort to convert existing translations.

**Comment by qsniyg** [comment]
Yeah, this was initially due to https://github.com/qsniyg/maxurl/issues/90. Thanks for reminding me, I'll get to fixing this.
